salt/utils/gitfs.py

```diff
@@ -458,46 +456,29 @@ def __init__(
             failhard(self.role)
 
         hash_type = getattr(hashlib, self.opts.get("hash_type", "md5"))
-        # Generate full id.
-        # Full id helps decrease the chances of collections in the gitfs cache.
-        try:
-            target = str(self.get_checkout_target())
-        except AttributeError:
-            target = ""
-        self._full_id = "-".join(
-            [
-                getattr(self, "name", ""),
```

Comment:
I see you are pretty much dropping full_id. however i don't see name being picked up elsewhere. are you still accounting for this functionality?

Reply:
yea self._full_id can be completely removed

Reply:
yup 100% fine to remove full_id